storage/src/main/java/com/tencent/rss/storage/api/ShuffleReader.java

```diff
@@ -26,5 +26,7 @@
 
   byte[] readData(long offset, int length);
 
+  byte[] readAllData();
+
   List<FileBasedShuffleSegment> readIndex(int limit) throws IOException, IllegalStateException;
```

Comment:
does this api needed?

Reply:
It's used in AbstractHdfsClientReadHandler and MultiStorageHdfsClientReadHandler, I will not make any change in this pr. Will tidy this interface in the following pr to refactor reading in multi mode.